packages/okta-react-native/index.js

```diff
@@ -0,0 +1,86 @@
+import { NativeModules, Platform } from 'react-native';
+import { assertIssuer, assertClientId, assertRedirectUri } from '@okta/configuration-validation';
+import jwt from 'jwt-lite';
+
+export const createConfig = async ({
+  clientId,
+  redirectUri, 
+  endSessionRedirectUri, 
+	discoveryUri, 
+	scopes
+}) => {
+
+  assertIssuer(discoveryUri);
+  assertClientId(clientId);
+  assertRedirectUri(redirectUri);
+  assertRedirectUri(endSessionRedirectUri);
+
+	if (Platform.OS === 'ios') {
+		scopes = scopes.join(' ');
+	}
+    
+  return NativeModules.OktaSdkBridge.createConfig(
+    clientId,
+    redirectUri,
+    endSessionRedirectUri,
+    discoveryUri,
+    scopes
+  );
+} 
+
+export const signIn = async () => {
+  return NativeModules.OktaSdkBridge.signIn();
+}
+
+export const signOut = async () => {
+  return NativeModules.OktaSdkBridge.signOut();
+}
+
+export const getAccessToken = async () => {
+  return NativeModules.OktaSdkBridge.getAccessToken();
+}
+
+export const getIdToken = async () => {
+  return NativeModules.OktaSdkBridge.getIdToken();
+}
+
+export const getUser = async () => {
+  return NativeModules.OktaSdkBridge.getUser();
+}
+
+export const getUserFromIdToken = async() => {
```

Comment:
Is `async` really required for the fast operations? Can it be just blocking calls?

Reply:
i'm using async here since the function still has to get the id token from the native modules, and that is an async operation :p

Reply:
Just thinking out loud.  I don't know too much about js world, but why we just can't get all required data once from native code and then just keep it in js part. I mean we can serialize access token, id token and etc. and return to js part during `signIn` call. Then we can get rid of async bridging functions, like getIdToken and getAccessToken

Reply:
That's a good idea to to think about, but I also had concerns with security if we take this approach. Since on native modules the tokens are stored securely. If we expose it to JS then JS has no way to store those securely.